ansible/vars/csm_packages.yml

```
@@ -66,12 +67,14 @@ application_csm_sles_packages:
   - cray-uai-util
   - smart-mon
   - smartmontools
+  - cray-spire-dracut>=2.0.0
 
 # All Compute nodes
 compute_csm_sles_packages:
   - acpid
   - bos-reporter
   - cray-uai-util
+  - cray-spire-dracut>=2.0.0
```

Same deal here too.

Again, no need to take this suggestion if this is the desired behavior.

```suggestion
  - cray-spire-dracut
```